### ISSUE TITLE
Documentation for the Review process

### ISSUE THREAD

**Issue by tobie** [w3c/testtwf-website#13]
E.g. based on the WebApps wiki.

**Comment by hubenjamin** [comment]
PR #37, commit hubenjamin@d48442935787af7bc2bf84713ddabea26b63c1be, and based on WebApp's workflow http://www.w3.org/wiki/Webapps/Submitting_tests#Test_Review_and_Approval_Workflow.

**Comment by hubenjamin** [comment]
Updated the link according to Tobie's review. Now commit hubenjamin@7573d24719884ca2ee3dadfdf070846bcb7d32ed.

**Comment by tobie** [comment]
Thanks. Don't bother updating the issue with what you add to the PR. WE get
notified anyway. Busy till Tuesday. Will look at this stuff then if someone
else hasn't in the meantime.

What I skimmed through looked nice. Seems that you're getting the hang of
this github/markdown thing! :-)

Sent from my mobile phone.

On Jun 6, 2013, at 18:44, hubenjamin notifications@github.com wrote:

Updated the link according to Tobie's review. Now commit
hubenjamin@7573d24https://github.com/hubenjamin/testtwf-website/commit/7573d24719884ca2ee3dadfdf070846bcb7d32ed
.

—
Reply to this email directly or view it on
GitHubhttps://github.com/w3c/testtwf-website/issues/13#issuecomment-19057768
.
